docs/slot-filling.md

```diff
@@ -128,4 +128,62 @@ main();
 // }
 ```
 
+## Entities with the same name
+
+Utterances with more than one entity with the same name are supported by providing "numbered entities".
+
+The "numbered entity" format must be in the form `${entityName}_${integer}`. E.g. "hero\_1", "food\_2", etc.
+
+```javascript
+const { NerManager } = require('node-nlp');
```

Comment:
The code uses NlpManager instead of NerManager, so NlpManager should be imported

Reply:
Doc updated.